### DISCUSSION TITLE
In PK-Sim, can we rely on Vss to explain the overprediction of MRT in IV and Oral simulations?

### DISCUSSION THREAD

**Discussion by sajabzughoul** [Open-Systems-Pharmacology/Forum#1055] [Questions & Problems]
in PK-Sim, can we rely on Vss to explain the overprediction of MRT ?

**Comment by tobiasK2001** [comment]
Could you please elaborate what you are up to? Why do you think the MRT is overpredicted?

**Comment by sajabzughoul** [comment]
I am working on AFE for PK parameters and the MRT AFE was 4
By simulating non-renal drug under renal disease

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by tobiasK2001** [comment]
hm, what is the AFE for the other PK parameters? Actually I would first look at the simulated plasma profiles vs measured data to get a first impression instead of focussing on single PK parameters.
